docs/getting-started/advanced-setting.asciidoc

```diff
@@ -67,6 +67,18 @@ IMPORTANT: {es-sec} requires {ecs-ref}[ECS-compliant data]. If you use third-par
 collectors to ship data to {es}, the data must be mapped to ECS.
 <<siem-field-reference>> lists ECS fields used in {es-sec}.
 
+[discrete]
+[[update-threat-intel-indices]]
+== Update default Elastic Security threat intelligence indices
+
+The `securitySolution:defaultThreatIndex` advanced setting defines threat intelligence indices that {elastic-sec} will use when collecting threat match data. One or more threat intelligence indices can be defined; the `filebeat-*` index is specified by default.
+
+The `securitySolution:defaultThreatIndex` setting controls features that query threat indices, such as the Threat Intelligence view on the Overview page and threat indicator match rules. You can enable Cross Cluster Search (CSS) by adding threat indices from remote clusters.
+
+NOTE: Indicator match rules typically run more queries than other rule types, which may affect rule performance.
```

Comment:
I don't think this note applies here; it should go under the Support section for IM rules: https://www.elastic.co/guide/en/security/current/detection-engine-overview.html#support-indicator-rules

Reply:
Removed for now. Will add to the [Limited support for indicator match rules](https://www.elastic.co/guide/en/security/7.12/detection-engine-overview.html#support-indicator-rules) section once a decision is made to update/not update info about indicator match rule performance.